aspnet/mvc/controllers/testing/sample/TestingControllersSample/tests/TestingControllerSample.Tests/project.json

```diff
@@ -0,0 +1,23 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "TestingControllerSample.Tests Class Library",
+  "authors": [ "Steve Smith (@ardalis)" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
```

Comment:
Test projects shouldn't need any of these metadata attributes (description, authors, tags, projectUrl, licenseUrl) - should just delete them.